### ISSUE TITLE
(GitHub Toggle Issue Comments) Reference messages are not listed/hidden

### ISSUE THREAD

**Issue by darkred** [Mottie/GitHub-userscripts#28]
Using 'GitHub Toggle Issue Comments' script version 1.1.2
with GM 3.17 in Firefox 56.01

Example link:
`https://github.com/Microsoft/vscode/issues/11847`
There are 3 such entries in that page (one is a group with 3 entries, the other two are single) .

I quote from the script code:
```
refs: {
	isHidden: false,
	name: "ghic-refs",
	selector: ".discussion-item-ref, .discussion-item-head_ref_deleted",
	label: "References"
},
```
Unfortunately the above selector `.discussion-item-ref, .discussion-item-head_ref_deleted` no longer works:
these entries now have a single class name instead, the plain: `.discussion-item`, as I noticed.
This class name is also given (among others) to other entries.
@Mottie, if that helps you, I *think* that a simple way to identify such entries is to check if their `.className === 'discussion-item' && .classList.length === 1` .

---

_PS. Btw, I've checked most other entry times (in other issue pages) and they all still work ok (I didn't find appropriate issue pages to check the: Integrations and the 'Diff current/outdated' entries)_

**Comment by Mottie** [comment]
Hey @darkred!

Thanks for letting me know. That does appear to be a recent change in their layout.

I recently added the functionality to check if the `.discussion-item` contains a certain class, so I'll have it check for the `h4` with a class of `.discussion-item-ref-title`, which appears to cover the two layouts I've seen in both the references from Feb 9 and Apr 4 ([ref](https://github.com/Microsoft/vscode/issues/11847#issuecomment-277808538)).

```js
refs: {
  isHidden: false,
  name: "ghic-refs",
  selector: ".discussion-item",
  contains: ".discussion-item-ref-title",
  label: "References"
},
```

Sadly, I don't remember where I got the `.discussion-item-head_ref_deleted` selector from, or if it still exists.
